File: app/src/main/java/de/markhaehnel/rbtv/rocketbeanstv/events/BufferUpdateEvent.java
package de.markhaehnel.rbtv.rocketbeanstv.events;

import lombok.Getter;

@Getter
public class BufferUpdateEvent {
    private BufferState status;
    public static int BUFFERING_PROGRESS = 0;
    public static int BUFFERING_END = 1;

    public enum BufferState {
        BUFFERING_PROGRESS,
        BUFFERING_END
    }

    public BufferUpdateEvent(BufferState status) {
        this.status = status;
    }
}

File: app/src/main/java/de/markhaehnel/rbtv/rocketbeanstv/events/BufferUpdateEvent.kt
package de.markhaehnel.rbtv.rocketbeanstv.events

class BufferUpdateEvent(val status: BufferUpdateEvent.BufferState) {

    enum class BufferState {
        BUFFERING_PROGRESS,
        BUFFERING_END
    }

    companion object {
        var BUFFERING_PROGRESS = 0
        var BUFFERING_END = 1
    }
}
